attestation-service/src/lib.rs

```diff
@@ -219,7 +219,7 @@ impl AttestationService {
     }
 
     /// Query Reference Values
-    pub async fn query_reference_values(&self) -> Result<HashMap<String, Vec<String>>> {
+    pub async fn query_reference_values(&self) -> Result<HashMap<String, serde_json::Value>> {
```

Comment:
Note that we are returning a struct `Value` not provided by `attestation_service` crate now. The user might meet situations where `serde_json` is not included in the code base and `serde_json::Value` is not found.

A workaround is to add `pub use serde_json::Value` to all the files that uses this struct.

A better solution is to wrap `serde_json::Value` into a new struct defined in rvps crate and implement some basic conversion logic from/into `serde_json::Value`.

It's ok for me to perfect it later with way 2 and just use way 1 at first.

Reply:
OK I added `pub use serde_json::Value` to the RVPS and AS libs.